web/ui/react-app/src/pages/rules/Rules.tsx

```diff
@@ -1,15 +1,147 @@
+          return (
+            <Table size="sm" striped bordered key={i}>
+              <thead>
+                <tr>
+                  <td colSpan={3}>
+                    <a href={'#' + g.name}>
+                      <h2>{g.name}</h2>
+                    </a>
+                  </td>
+                  <td>
+                    <h2>{formatRelative(g.lastEvaluation, now())} ago</h2>
+                  </td>
+                  <td>
+                    <h2>{roundUp(parseFloat(g.evaluationTime) * 1000)}ms</h2>
+                  </td>
+                </tr>
+              </thead>
+              <tbody>
+                <tr className="font-weight-bold">
+                  <td>Rule</td>
+                  <td>State</td>
+                  <td>Error</td>
+                  <td>Last Evaluation</td>
+                  <td>Evaluation Time</td>
+                </tr>
+                {g.rules.map((r, i) => {
+                  return (
+                    <tr key={i}>
+                      {r.alerts ? (
+                        <td>
+                          <strong>alert:</strong>
+                          <Link
+                            className="ml-4"
+                            to={encodeURI(
+                              `${pathPrefix}/new/graph?g0.expr=ALERTS{alertname="${r.name}"}&g0.tab=1&g0.stacked=0&g0.range_input=1h`
+                            )}
+                          >
+                            {r.name}
+                          </Link>
+                          <br />
+                          <span className="rules-head">expr:</span>
+                          <Link
+                            className="ml-4"
+                            to={encodeURI(
+                              `${pathPrefix}/new/graph?g0.expr=${r.query}&g0.tab=1&g0.stacked=0&g0.range_input=1h`
+                            )}
+                          >
+                            {r.query}
+                          </Link>{' '}
+                          <br />
+                          <AlertRuleMaps map={r.labels} term="labels" />
+                          <AlertRuleMaps map={r.annotations} term="annotations" />
+                        </td>
+                      ) : (
+                        <td>
+                          <span className="rules-head">record:</span> {r.name} <br />
+                          <span className="rules-head">expr:</span>
+                          <Link
+                            className="ml-4"
+                            to={encodeURI(
+                              `${pathPrefix}/new/graph?g0.expr=${r.query}&g0.tab=1&g0.stacked=0&g0.range_input=1h`
+                            )}
+                          >
+                            {r.query}
+                          </Link>
+                        </td>
+                      )}
+                      <td>
+                        <Alert className="d-inline-block p-1 text-uppercase">{r.health.toUpperCase()}</Alert>
```

Comment:
In the new UI on the /targets page we have started using badges instead of alerts for these tiny health status texts. I would do the same here too. See: https://github.com/prometheus/prometheus/blob/2b2eb79e8b7245d2d60001ae08443d31f7b3a5b9/web/ui/react-app/src/pages/targets/ScrapePoolPanel.tsx#L65